src/app/authorization/login/login.component.css

```diff
@@ -1,13 +1,17 @@
-*{
-    font-family: Roboto;
-}
-h6{
-    margin-top: 14px;
-}
-.card{
-    margin: 0 auto; /* Added */
-    /* float: none; Added */
-    /* margin-bottom: 30px; Added */
-    border-radius: 15px;
-    height: auto;    
-}
\ No newline at end of file
+* {
+  font-family: Roboto;
+}
+
+label {
+  font-size: calc(70% + 1vh);
+}
+
+.form-text {
+  font-size: calc(70% + 1vh);
```

Comment:
here calc will not work as ther is no scss file so calculation will be failed could you please check

Reply:
yes, i checked it and i changed the file from css to scss.